roofit/roofitcore/src/RooAbsPdf.cxx

```diff
@@ -956,14 +956,21 @@ RooAbsReal* RooAbsPdf::createNLL(RooAbsData& data, const RooLinkedList& cmdList)
   const char* globsTag = pc.getString("globstag",0,kTRUE) ;
   Int_t ext      = pc.getInt("ext") ;
   Int_t numcpu   = pc.getInt("numcpu") ;
-  RooFit::MPSplit interl = (RooFit::MPSplit) pc.getInt("interleave") ;
+  Int_t numcpu_strategy = pc.getInt("interleave");
+  // strategy 3 works only for RooSimultaneus.
+  if (numcpu_strategy==3 && !this->InheritsFrom("RooSimultaneous") ) {
+      coutW(Minimization) << "Cannot use a NumCpu Strategy = 3 when the pdf is not a RooSimultaneus, "
+                           "use then the default strategy = 0" << endl;
```

Comment:
```suggestion
                           "falling back to default strategy = 0" << endl;
```

Reply:
Thank you for the suggestion. 
I have updated the commit